resources/views/admin/configurations/index.blade.php

```diff
@@ -5,8 +5,11 @@
 
         @can('admin.configurations.write')
             <div class="d-flex justify-content-end my-3">
-                <a href="{{route('admin.configurations.create')}}" class="btn btn-primary"><i
+                <a href="{{route('admin.configurations.create')}}" class="btn btn-primary mx-2"><i
                         class="fa fas fa-cog pe-2"></i>{{__('Create configuration')}}</a>
+
+                <a href="{{route('admin.configurations.sync')}}" class="btn btn-primary mx-2"><i
```

Comment:
The button placement is a bit awkward; if a new user hits this button, it will do something but not show what it has done or has been added/changed to the panel. Only if you know what syncing does and means do you know the purpose of this button. I think it might be better to have this on an overview page or add it to the create configuration page as a small link below the linking part. Like "New Nodes or Eggs not showing? Sync you pterodactyl with controlpanel"

Reply:
yeah ok ill change that up tomorrow and let you know if its stable or not